iris/src/iris/domain/status/global_search_status_update_dto.py

```
@@ -16,4 +14,4 @@ class GlobalSearchStatusUpdateDTO(StatusUpdateDTO):
 
     result: Optional[str] = None
     answer: Optional[str] = None
-    sources: List[LectureSearchResultDTO] = Field(default_factory=list)
+    sources: List[GlobalSearchSourceDTO] = []
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)